snapcraft.yaml

```diff
@@ -41,7 +41,7 @@ description: |
                target: ubuntu-app-platform
          and then make your apps use it:
            plugs: [platform]
-       - similarly for the gnome runtime
+       - similarly for the gnome platform based apps
           plugs:
               gnome-runtime:
```

Comment:
use platform here (and below) as we are going to change that to the platform snap
Please change it in the helper as well with backward compat (test the dir existance)